certified-connectors/BuildingMindsTwin/apiProperties.json

```diff
@@ -0,0 +1,37 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "required",
+          "scopes": [
+            "https://buildingminds.onmicrosoft.com/cd1b0fa3-bc0f-4cd7-a09c-dbaf6c19db40/user_impersonation openid offline_access"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "redirectUrl": "https://global-test.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False"
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://buildingminds.b2clogin.com/buildingminds.onmicrosoft.com/B2C_1A_SignInUnified/oauth2/v2.0/authorize"
+            },
+            "tokenUrl": {
+              "value": "https://buildingminds.b2clogin.com/buildingminds.onmicrosoft.com/B2C_1A_SignInUnified/oauth2/v2.0/token"
+            },
+            "refreshUrl": {
+              "value": "https://buildingminds.b2clogin.com/buildingminds.onmicrosoft.com/B2C_1A_SignInUnified/oauth2/v2.0/token"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#428e92",
+    "capabilities": [],
+    "publisher": "BuildingMinds",
```

Comment:
@duncanbinger Does your company name spells with out a space between 'Building' and 'Minds' ?

Reply:
@sriyen-msft - Correct. The company name is without a space. BuildingMinds. The legal entity is BuildingMinds GmbH